website/docs/guides/guides-upgrade.md

```diff
@@ -4,39 +4,39 @@ title: Upgrade to a newer version
 sidebar_position: 1
 ---
 
-## Foreword
-
-> Please note that this guide is work in progress!
-> If there is one step you should follow one by one, then it is the backup of the current system.
-> If you find any issues with the guide, then please tell us so (by [opening a new issue on Github](https://github.com/lansuite/lansuite/issues/new))!
-
 ## Prerequisites
 
-* A system having both Git and Composer up and running if you want to use the Git Master
-* A current file and database backup
-* Your target system running PHP >= 7.0 and MySQL >= 5.6.3
+* Your target system need to run PHP >= 8.1 and MySQL >= 5.7
 * A fair bit of time
 
 ## Preparation
 
-### Plan
+### Download an official release
 
-It is recommended to tell your users (and fellow administrators) in advance that your LanSuite instance won't be 100% functional available during the upgrade.
-That should be well done in advance if you run a larger installation, for smaller ones no one may notice. But better be safe than sorry.
+1. Download the release from [GitHubs Releases page](https://github.com/lansuite/lansuite/releases)
+2. Extract the archive on your machine
 
-### Prepare Upgrade files
+### Execute a backup
```

Comment:
The order of operation is wrong.
The system should be locked first and then the backup created.
If you do it the other way around then there is a risk that additional changes are done that are missing from the backup.
_Especially_ if you also do not tell other admins that you are doing an upgrade ;)

Reply:
Fixed in https://github.com/lansuite/lansuite/pull/794/commits/a1a9aebebca65f92766cf496a36a0da3b311b7a2